doc/source/deploy-on-yarn.rst

```diff
@@ -130,51 +125,14 @@ Clean up all started processes even if the application fails or is killed.
 .. code-block:: bash
 
     ray stop
-    skein application shutdown $APP_ID
+    skein application shutdown current
 
 Putting things together, we have:
 
-.. code-block:: bash
-
-    services:
-        ray-head:
-            # There should only be one instance of the head node per cluster.
-            instances: 1
-            resources:
-                # The resources for the head node.
-                vcores: 1
-                memory: 2048
-            files:
-                # ray/doc/yarn/example.py
-                example.py: example.py
-            #     # A packaged python environment using `conda-pack`. Note that Skein
-            #     # doesn't require any specific way of distributing files, but this
-            #     # is a good one for python projects.
-            #     # See https://jcrist.github.io/skein/distributing-files.html
-            #     environment: environment.tar.gz
```

Comment:
*Unfortunately github diff commenting only lets me comment on changes, so I'll add this here*

This section is copied around in a few places, and may be confusing for new readers as it's double commented out, and not really explained. I'd either drop it, or alter the language to make it clearer that including a packaged  environment file is optional, and if they wish to do so they can do so here in this way.

Also, you may want to remove the code block from https://github.com/ray-project/ray/pull/6173/files#diff-de6110281d163f755df4b298ab7b2bc6R71 and use a `literalinclude` to the example file instead, not sure.